bubblewrap.c

```diff
@@ -1980,7 +2023,10 @@ parse_args_recurse (int          *argcp,
           if (next_perms >= 0)
             op->perms = next_perms;
           else
-            op->perms = 0666;
+            if (opt_compat_level == 0)
+              op->perms = 0666;
+            else
+              op->perms = 0644;
```

Comment:
Would 0600 be a better new default for `--file` than 0644, in the spirit of "failing safe" if the file might contain secrets? That would make it consistent with `--[ro-]bind-data` in compat level >= 1.

Reply:
If you do that, then the documentation for `--[ro-]bind-data` should change to something like "note that this is not the same as --file in compatibility levels older than 1".